django_ninja_crudl/endpoints/create.py

```
@@ -1,9 +1,12 @@
 """CRUDL API base class."""
 
+import weakref
 from abc import ABC
 from typing import TYPE_CHECKING, Literal, Unpack
 
 from django.db import transaction
+from django.db.models.signals import ModelSignal, post_save, pre_save
```

Pylint (3.3.8)

[error] Unable to import 'django.db.models.signals'

(E0401)